plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSplit.java

```diff
@@ -168,6 +173,16 @@ public String getPartitionSpecJson()
         return partitionSpecJson;
     }
 
+    /**
+     * Trino (stack) values of the partition columns. The values are the result of evaluating
```

Comment:
What does `(stack)` mean here ?

Reply:
stack values are long, double, boolean and object, where as actual values might be something like float